04-gas/gas-section-three/gas-section-three.html

```diff
@@ -27,15 +28,19 @@ <h1 class="gas-section-three-title">
               on your gas bill. The more space between furniture and radiators the better.
```

Comment:
There should be two separate Paragraphs per the Figma.
The second one starts with "This can save"

Reply:
done